### PR TITLE
Partial implementation of the homepage. More updates pending

### PR DESCRIPTION
**Pull request by tims139**

### CODE REVIEW

**Comment by tims139** [comment]
It might be necessary to make it look nicer for the mobile version

File: .env
Comment (akhilwaghmare) [inline .env:1]: This should not be tracked by git. Make sure you merge in the latest `main`, which should include `.env` in the .gitignore file

**Review by akhilwaghmare — CHANGES_REQUESTED** [review]
Can you merge in the latest `main` branch to make sure you have the latest `.gitignore`? Other `.env` is being tracked, it should not be

File: .env
Comment (tims139) [reply .env:1]: Done!

**Review by akhilwaghmare — APPROVED** [review]

**Pull request merged**